src/init/features/functions/index.js

```diff
@@ -22,14 +21,14 @@ module.exports = function(setup, config) {
 
   setup.functions = {};
   var projectId = _.get(setup, "rcfile.projects.default");
-  var enableApis;
+  var enableApis = Promise.resolve();
   if (projectId) {
-    enableApis = requireAccess({ project: projectId }, [scopes.CLOUD_PLATFORM]).then(function() {
-      enableApi(projectId, "cloudfunctions.googleapis.com");
-      enableApi(projectId, "runtimeconfig.googleapis.com");
+    enableApis = requirePermissions({ project: projectId }).then(() => {
```

Comment:
The only difference here is that `scopes.CLOUD_PLATFORM` is no longer being checked.

Reply:
Which is fine because we require CLOUD_PLATFORM for everything now anyway.